lib/aws/src/policy.ts

```diff
@@ -59,7 +59,7 @@ export const deletePolicy = ({
   PolicyArn
 }: {
   PolicyArn: string;
-}): Promise<IAM.Policy | undefined> => {
+}): Promise<void> => {
```

Comment:
`resolve()` is what we do below. The type signature didn't reflect that, right? (why did this compile?)